tests/Aquifer.Public.API.IntegrationTests/AppFixture.cs

```diff
@@ -0,0 +1,66 @@
+﻿using FastEndpoints.Testing;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Aquifer.Public.API.IntegrationTests;
+
+/// <summary>
+/// This is a FastEndpoints <see cref="AppFixture"/> that sits on top of a <see cref="WebApplicationFactory{TEntryPoint}"/>
+/// which allows for in-memory web requests to be sent and received without actual network traffic.
+/// Details:
+/// * https://fast-endpoints.com/docs/integration-unit-testing
+/// * https://learn.microsoft.com/en-us/aspnet/core/test/integration-tests
+/// </summary>
+public sealed class AppFixture : AppFixture<Program>
+{
+    /// <summary>
+    /// Configure Clients here.
+    /// The default Client is anonymous and has no API Key header value.
+    /// That is sufficient for Public API tests because all routes are anonymous and no actual web requests are sent via this fixture.
```

Comment:
Probably worth adding the point that the API key is enforced by Azure API Management, and that's why it's not needed locally.